src/main/kotlin/org/rust/ide/intentions/AbstractListIntention.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.psi.PsiComment
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.ext.RsElement
+import org.rust.lang.core.psi.ext.leftSiblings
+import org.rust.lang.core.psi.ext.rightSiblings
+
+abstract class AbstractListIntention<TList : RsElement, TElement : RsElement>(
```

Comment:
We usually use `SomeSemanticNameBase` name convention for abstract classes

Reply:
Fixed.